packages/tags/src/Tags.ts

```diff
@@ -70,7 +70,6 @@ export class Tags extends FocusVisiblePolyfillMixin(SpectrumElement) {
     private handleKeydown = (event: KeyboardEvent): void => {
         const { code } = event;
         if (code !== 'PageUp' && code !== 'PageDown') return;
-
```

Comment:
Can we clean up this white space change?

Reply:
done